### PR TITLE
rules added for OCP-25816

### PR DESCRIPTION
**Pull request by yapei**
@SSshahan   Can you help review?

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/goto.xyaml [hunk 1]
```diff
@@ -99,6 +99,10 @@ goto_catalog_page:
 goto_create_app_from_imagestream_page:
   url: catalog/source-to-image?imagestream=<is_name>&imagestream-ns=openshift&preselected-ns=<project_name>
 
+# monitoring
+goto_monitoring_metrics_page:
+  url: monitoring/query-browser?query0=
+
```
Comment (SSshahan) [inline lib/rules/web/admin_console/4.3/goto.xyaml]: How about `monitoring/query-browser` for the url?

**Comment by SSshahan** [comment]
Others looks good to me.thx

File: lib/rules/web/admin_console/4.3/goto.xyaml (hunk 1 above)
Comment (yapei) [reply lib/rules/web/admin_console/4.3/goto.xyaml]: visiting `monitoring/query-browser` will redirect to `monitoring/query-browser?query0=`. I think I can change it to a simpler one

Comment (yapei) [reply lib/rules/web/admin_console/4.3/goto.xyaml]: updated

**Comment by yapei** [comment]
@akostadinov  @pruan-rht  Can you help review and merge? Thanks

**Pull request merged**